lib/services/osmdata.dart

```diff
@@ -343,36 +345,40 @@ class OsmData{
 
       var routeAlternativeNodes = List<Node>();
       routeAlternative.forEach((edge) => routeAlternativeNodes.addAll(graph.edgeToNodes(edge)));
-      result.add(routeAlternativeNodes);
+      result.add(HikingRoute(routeAlternativeNodes, lengthOfEdgesKM(routeAlternative), List()));
```

Comment:
is it possible to overload the constructor so that it is not neccessary to pass a empty list when creating a route? Or will there always be pois?

Reply:
done